api/sft/handler.go

```diff
@@ -27,11 +28,25 @@ func NewHandler(r chi.Router, s *Service) *Handler {
 
 func StartDashboard(port string) {
 
-	// TODO: sort out path to the dashboard
-	fs := http.FileServer(http.Dir("/Users/maxbb/github/revitt/sft/web/dashboard"))
-	http.Handle("/", fs)
+	//TODO: sort out path to the dashboard - this needs to be managed by a config when the dashboard is ready
+	buildPath := "/Users/maxbb/github/revitt/sft/web/dashboard/dist"
+
+	fs := http.FileServer(http.Dir(buildPath))
+
+	// this handler function is designed to allow us to serve a react app with react router
```

Comment:
I guess react router is inconsequential in backend talk. I kinda get why the comment is here, but feel it could mislead you!

Reply:
Yeah, it was there to remind me really! I've updated it (again it's mainly for me I guess) but do you think it should be deleted? I'd been keen to know what you'd do there (e.g. do you inherently just know what's going on from the code?)